QEfficient/utils/_utils.py

```diff
@@ -61,16 +62,21 @@ def hf_download(
                 ignore_patterns=ignore_patterns,
             )
             break
+        except GatedRepoError as e:
+            raise RuntimeError(
+                f"Looks like you are trying to access gated model from HuggingFace, please make sure you have access and pass hf_token\n{e}"
+            )
         except requests.ReadTimeout as e:
             logger.info(f"Read timeout: {e}")
             retry_count += 1
-
         except HTTPError as e:
             retry_count = max_retries
             if e.response.status_code == 401:
                 logger.info("You need to pass a valid `--hf_token=...` to download private checkpoints.")
             else:
                 raise e
+        except Exception as e:
+            raise e
```

Comment:
This is not needed, by default uncaught exception is always raised.

Reply:
Try this:

```python
# except_test.py
try:
    raise OSError("ose")
except FileNotFoundError:
    print("fnfe")
except RuntimeError:
    print("rte")
```

You'll get:

```
Traceback (most recent call last):
  File "except_test.py", line 3, in <module>
    raise OSError("ose")
OSError: ose
```

Reply:
Okay, will remove. Thanks